src/content/en/2021/css.md

```diff
@@ -590,28 +672,39 @@ featured_stat_label_3: TODO
   sql_file="image_formats.sql"
 ) }}
 
+We did not attempt to determine whether any of the images were animated.
 
+Please also note that this analysis only covers the images loaded by CSS: we did not check the HTML to see what was being loaded there.  Thus, the following results cannot be taken as a metric of how heavy web pages are, or even how heavy CSS is or is not.  It can only show how much CSS-loaded images contribute to a page’s total weight.
+
+
+### Number of images in CSS
+
+What we found was that most CSS doesn’t result in a lot of image loads: the lower two percentiles came in at one image each, and even the 90th percentile hovered around 10 images, across all image types.
 
 {{ figure_markup(
   image="number-of-images-loaded.png",
-  caption="Distribution of the number of external images loaded via CSS.",
+  caption="Distribution of the median number of external images loaded via CSS.",
```

Comment:
Could you explain your reasoning for including the word "median" here? The distribution is summarized by the percentiles, one of which is the median, but I wouldn't say that it's a "distribution of the median".
```suggestion
  caption="Distribution of the number of external images loaded via CSS.",
```